casper/src/test/scala/coop/rchain/casper/util/rholang/InterpreterUtilTest.scala

```diff
@@ -303,17 +313,29 @@ class InterpreterUtilTest
     pendingUntilFixed { //reference costs
       //deploy each Rholang program separately and record its cost
       val deploy1 =
-        ProtoUtil.termDeploy(mkTerm("@1!(Nil)").toOption.get, System.currentTimeMillis())
+        ProtoUtil.termDeploy(
+          mkTerm("@1!(Nil)").toOption.get,
+          System.currentTimeMillis(),
```

Comment:
In general it's not safe to use the current time in tests. We could get to situations when the tests fail because they happened to run at the wrong moment.

Reply:
It's how they've been for quite some time now and they seem to work. Let's leave it as it is for now. It's not the change introduced in this PR.

Reply:
ok